### DISCUSSION TITLE
Release Versioning Causes Confusion

### DISCUSSION THREAD

**Discussion by RacerDelux** [dsbenghe/Novell.Directory.Ldap.NETStandard#256] [General]
I was trying to figure out why I was not seeing the new functions added in beta 5.
I pulled down the repo and figure out why I think.

The tag for version 5 is before the changes were committed. Is this intended?
When I include the nuget package, I don't see any of the commits after the v5 tag.
![image](https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/assets/2667344/ab9c0ea7-3cdd-496e-9792-e769366e4c79)

**Comment by dsbenghe** [comment]
there were further changes after the beta5 release which were not yet part of any release.

**Comment by RacerDelux** [comment]
Oh gotcha. Maybe I just got confused by GitHub. It kept marking issues resolved and linking it to the beta 5 release.
A little off topic, but question.
How come in some of the new changes - support for passing the cancellation token is being dropped?

**Reply by TheConstructor** [reply]
Any example were you think it would have been appropriate? I can only guess, that it previously had no effect, and now that fact is more easily visible. In the end many people assume, that arguments are evaluated/used and may not check with documentation or code, if that is true
